internal/stm32CubeMX/stm32CubeMX.go

```diff
@@ -135,6 +151,10 @@ func Process(cbuildYmlPath, outPath, cubeMxPath string, runCubeMx bool, pid int)
 		for {
 			proc, err := os.FindProcess(pid) // this only works for windows as it is now
```

Comment:
Is this comment true? How does it work on Linux and macOS then?

Reply:
There is an additional proc.Signal(syscall.Signal(0)) which correctly checks for existing process